entry_types/scrolled/package/spec/frontend/MediaPlayer-spec.js

```diff
@@ -84,6 +84,26 @@ describe('MediaPlayer', () => {
       expect.anything()
     );
   });
+  
+  it('it passes correct context data to getPlayer options ', () => {
+    render(<MediaPlayer {...requiredProps()}
+                        type={'video'}
+                        sources={getVideoSources()} />);
+
+    expect(media.getPlayer).toHaveBeenCalledWith(
+      expect.anything(),
+      expect.objectContaining({
+        mediaEventsContextData: {
+          page: {
+            configuration: {
+              title: undefined
```

Comment:
Consider passing a specific `mediaEventsContext` to have a more eloquent test that clarifies the format that `MediaPlayer` expects. This has the advantage that even in the absence of TypeScript etc. other developers can figure out the interface more quickly. Also, this way, you can test for more realistic context data